wordpress/manifest-int.yml

```diff
@@ -8,7 +8,7 @@
 ---
 applications:
 - name: dceas-admin-site
-  buildpack: https://github.com/cloudfoundry/php-buildpack#v4.4.26

```

Comment:
We no longer enforce the buildpack version since the PHP issue is now fixed. It should use the most recent version by default now.